kernel/base/src/main/java/com/twosigma/beaker/chart/xychart/plotitem/XYGraphics.java

```diff
@@ -92,7 +92,7 @@ public void setX(List<Object> xs) {
   }
 
   public List<Number> getX() {
-    if (xs == null) {
+    if (xs == null || xs.isEmpty()) {
```

Comment:
why was this added?  doesn't seem right.  an empty xs array seems like an error.